### PR TITLE
SF-1121: Manage packages as workspaces in the monorepo

### PR DESCRIPTION
**Pull request by Vic-Dev**

### CODE REVIEW

File: packages/breadcrumbs/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+root = true
```
Comment (jmbtutor) [inline packages/breadcrumbs/.editorconfig]: Pull up

File: packages/breadcrumbs/package.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  "license": "MIT",
+  "main": "dist/index.js",
+  "module:esnext": "esnext/index.js",
+  "files": [
+    "dist/",
+    "esnext/",
+    "src/"
+  ],
+  "scripts": {
+    "prepare": "install-self-peers -- --ignore-scripts",
```
Comment (jmbtutor) [inline packages/breadcrumbs/package.json]: Unnecessary

File: packages/breadcrumbs/LICENSE [hunk 3]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 GroupBy Inc
```
Comment (Vic-Dev) [inline packages/breadcrumbs/LICENSE]: Update the year to `2018`

File: packages/breadcrumbs/package.json (hunk 2 above)
Comment (JoeyDeol) [reply packages/breadcrumbs/package.json]: Will be address in SF-1143

**Pull request merged**